docroot/modules/humsci/hs_courses/modules/hs_courses_importer/tests/Functional/HsCoursesImporterFormTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
```

Comment:
I see you are writing these as standard practice now?

Reply:
Yeah i'm gonna start doing more of these as i build out functionality. I want to start getting better at them and get the coverage report to increase. Unfortunately, Functional tests break the code coverage report. Since Functional tests install Drupal into a database, it causes xdebug and code coverage repot to go SUPER slow. thats why i removed the coverage report in this PR until I find a better solution/alternative.

Reply:
Like the test just above this failed, because i removed a method.. perfect use 👍